src/lightingCAN.cpp

```diff
@@ -1,22 +1,93 @@
 #include "lightingCAN.h"
 
+#define L_FRONT_LIGHT 0
+#define R_FRONT_LIGHT 1
+#define L_SIDE_LIGHT 2
+#define R_SIDE_LIGHT 3
+#define MID_LIGHT 4
+#define L_REAR_LIGHT 5
+#define R_REAR_LIGHT 6
+
+#define BOARD 0
+
```

Comment:
Might want to leave a comment saying that `BLINK == 0` means not blinking, and  `BLINK == 1` means blinking so others can quickly understand what the macros are for.